### DISCUSSION TITLE
Logo  design your input

### DISCUSSION THREAD

**Discussion by ynnebbenny** [probonopd/MiniDexed#278] [General]
Do we need a group banner? 
I'm trying to come up with a design to put onto a 3d printed case that Im also working on. Im wondering what improvements on this design you might suggest.
![minidexed](https://user-images.githubusercontent.com/100736840/173041102-418f4358-0749-4052-bb33-fd235b4db2f5.png)
![minidexedw](https://user-images.githubusercontent.com/100736840/173041112-a590a13f-63ed-4e21-83b6-db8f212448e6.png)

**Comment by sou7611** [comment]
I Iike the latter much better

**Comment by dcoredump** [comment]
> Do we need a group banner? 
> I'm trying to come up with a design to put onto a 3d printed case that Im also working on. Im wondering what improvements on this design you might suggest.
> ![minidexed](https://user-images.githubusercontent.com/100736840/172513921-09e0ae97-b27d-4397-b9dc-68b10c492918.png)
> 

Nice idea, but be careful because of the similarity to other banners...

**Comment by now-its-dark** [comment]
Hey there @ynnebbenny, I dig where you're going with this! One thing to be mindful of when using acute angles is the mitre limit setting of the stroke— it's visible at the inner angles of the outlined E in your design, because they don't have sharp points like they do on the outer edge. Also, the solid and outlined styles were originally only used for numerals, but there's of course room for creative license, just calling out that it's not to the spec 😄.

Thought it would be fun to riff on this and try some variants. Feel free to mess with or use anything! Experimenting with the later (1986-onward) type treatment they used, as it is a bit more flexible / visually balanced for longer strings of letters IMO, but also more restrained and a bit less iconic, so somewhat of a tradeoff.  

![minidexed Logo_page](https://user-images.githubusercontent.com/5728860/172754788-4e602bfc-b647-4416-b4a1-9c3f3875a69c.svg)
.

**Reply by ynnebbenny** [reply]
Great that you are having a play around too. Yeah that outlined E cut corners is a strange thing, Its not for the reason you mentioned, but yes on the fix list. From my point of view the end result will be a vector drawing for my 3d printed case. White text printed on a black case.
Keep at it , you've done some great designs.!!!

**Comment by probonopd** [comment]
Let's not use something that could be mistaken for a trademarked logo due to its visual similarity.

The open source software https://asb2m10.github.io/dexed/ is using this:

![image](https://user-images.githubusercontent.com/2480569/172903939-be5fc469-9581-4341-a91a-e2e6172c19ab.png)

**Reply by now-its-dark** [reply]
Yeah definitely too close for comfort for an official logo, can be cool for personal device customization / 3d printed shells though 😄.

**Reply by now-its-dark** [reply]
For an official logo, Chipsynth OPS7 (the dx7 emulation by Plogue) is probably the best reference here, as it is commercial / would have dialed in the legal aspect. Their logo designs are solid and do a good job of retaining the feel without being overly derivative.

**Reply by now-its-dark** [reply]
![csops7-exif-remove](https://user-images.githubusercontent.com/5728860/172907745-dda4ab2a-5e95-43e1-9b71-8b412b4b3faa.png)

**Comment by dazarz** [comment]
Hello!

I've come up with two new logo designs for MiniDexed. Which one do you prefer? 

v1:
![minidexed-logo-v1](https://github.com/user-attachments/assets/8bd48a06-3f67-45b3-8b81-294d9d988c28)


v2:
![minidexed-logo-v2](https://github.com/user-attachments/assets/8b408c1a-ab24-4231-a7f5-e88790efa14d)

**Reply by plamikcho** [reply]
My vote for V2. But the "mini" probably will become too small if the overall size is smaller.

**Comment by dazarz** [comment]
Another attempt. The word 'mini' was created using a free font, while 'dexed' was designed from scratch as a graphic.

v3
![minidexed-logo-v3](https://github.com/user-attachments/assets/07267878-6545-4f25-96ff-2eddddca1895)

v4
![minidexed-logo-v4](https://github.com/user-attachments/assets/ec79597a-b5cd-4ec8-9b61-8ae2b1759f9f)

**Reply by plamikcho** [reply]
v4 is better, v3 "mini" font looks a bit strange next to the other font.

**Comment by dazarz** [comment]
Probably the last attempt :)

v5
![minidexed-logo-v5](https://github.com/user-attachments/assets/26fcf099-88a5-4aa0-b963-ef8a5f52d306)

**Comment by Lonolo** [comment]
Here's my attempt,

<img width="674" alt="Screen Shot 2025-01-18 at 8 56 24 PM" src="https://github.com/user-attachments/assets/5ac95112-2f34-4bff-a369-b384476387bb" />
